### DISCUSSION TITLE
Thermostat VERSATIL  Questions

### DISCUSSION THREAD

**Discussion by cfrancky** [jmcollin78/versatile_thermostat#756] [Q&A]
Bonjour et pour débuter je souhaites a tous de bonnes fêtes de fin d'année ....

Ma configuration est simple, une pompe a chaleur eau/eau. une sonde Sonoff  SNZB-02D  et deux modules NODON SIN-4-1-21
un module pour la commande de la PAC et le deuxieme pour le pilotage de la pompe chauffage, en effet la configuration de base de la PAC faisait fonctionner la pompe chauffage H24 même si il n'y avait pas besoin de chauffage. J'ai donc souhaiter optimiser afin de faire des économie d'electricité.

Le chef d'orchestre etant le thermostat Versatil  

j'ai donc créer les deux 

![image](https://github.com/user-attachments/assets/97d6dab8-cfb1-4cc9-88c9-a37c5b386faf)

Donc une configuration centrale et un thermostat chauffage type "Thermostat sur un switch"
dans les paramètres "sous jacent" de mon thermostat chauffage j'ai bien en entité dans les équipements a contrôler
le switch de la PAC et également le switch de commande de la pompe chauffage.

![image](https://github.com/user-attachments/assets/de8c9f9d-f033-46a0-bcd6-09cfb31263cf)

Quand il y a demande de chauffage  du thermostat, la PAC est bien en demande mais pas la pompe chauffage
par contre en demande d'arrêt, tout semble fonctionner normalement la PAC et la pompe chauffage s'arrête bien, 
Bien sur pour arriver a cela je suis obligé de forcer la marche de la pompe chauffage pour quelle démarre......

Si vous avez des idées par avance merci

**Comment by jmcollin78** [comment]
Hello @cfrancky 

Quel seuil as-tu mis ?

<img width="276" alt="Capture d’écran 2024-12-27 à 21 44 17" src="https://github.com/user-attachments/assets/672f7324-d244-4054-8dd7-c0bc65dd5858" />

Cf. https://github.com/jmcollin78/versatile_thermostat/blob/main/documentation/fr/feature-central-boiler.md

**Comment by cfrancky** [comment]
Bonsoir, dans mon cas le thermostat régule sur la température ambiante et
pilote de relais Nodon qui ferme le contact marche PAC.

Je n'ai pas de regulation chaudiere, la PAC a juste besoin de
l'autorisation de marche.
Cela marche très bien le Versatil gère plutôt bien.
Mais je souhaite prendre la main sur la pompe chauffage afin que celle ci
fonctionne que si il y a une demande de chauffage.
Ainsi éviter qu'elle tourne en permanence.



Le ven. 27 déc. 2024, 21:45, Jean-Marc Collin ***@***.***> a
écrit :

**Comment by jmcollin78** [comment]
Hello,

Je pense qu'on ne se comprend pas (en tout cas je ne comprends pas ta réponse). VTherm peut piloter un switch de chaudière centrale (cf le lien que je t'ai donné ci-dessus). Il ne faut pas mettre le switch de la chaudière dans le VTherm.

**Comment by cfrancky** [comment]
Bonjour, effectivement mais je pense que je n'utilise pas obligatoirement
les bons termes.
J'ai bien lu la partie contrôle d'un chaudière centrale, mais
n'ayant qu'une sonde d'ambiance, j'ai donc utilisé
une configuration central sans gestion de chaudiere central et un Vtherm
qui lui contrôle l'équipement "switch chauffage".
jusque là tout marchait bien puisque la pompe chauffage était en service
H24 a partir du moment où je demandais le chauffage sur la PAC (inter
marche chauffage sur la PAC).
j'ai recablé mon système afin de permettre a HA de prendre en charge la
commande de la pompe chauffage et ainsi ne la faire fonctionner que quand
le Vtherm demandait la chauffe.
dans le Vtherm j'ai ajouté l'entité "marche pompe chauffage" après l'entité
marche chauffage (qui commande la PAC).

En te lisant je comprends que c'est pas ce qu'il faut faire.
Je dois indiquer a Versatile dans la configuration central que

[image: image.png]

ensuite indiqué les lignes de commande pour désigner le ou les switch a
activer ou désactiver quand un Vtherm est en demande de marché ou arret

[image: image.png]

bien sur je dois dans le Vtherm indique que celui si est utilisé pour
piloter une chaudière central

[image: image.png]

je vais pas plus loin pour le moment dans mes interrogation pour rester
clair (dans ma tete), mais est ce que je suis claire et surtout ok au
niveau raisonnement

Je rappel mon cahier des charge, en demande chauffage je dois activer deux
switch, la commande de la PAC et la commande de la pompe chauffage

Par avance merci

Le ven. 27 déc. 2024 à 22:36, Jean-Marc Collin ***@***.***> a
écrit :

**Comment by cfrancky** [comment]
Après plusieurs lectures du mode d'emploi, je suis parvenu à:
La configuration centrale pilote la marche de ma pompe a chaleur en
fonction de la demande du seul Vtherm.
le Vtherm appelé thermostat chauffage que le central configuration
surveille, envoi l'ordre de marche au switch de la pompe chauffage

Du coup cela fonctionne, peut être pas la méthode la plus pro mais bon
l'idéal aurait été que la configuration centrale pilote à la fois la PAC et
la pompe chauffage
